providers/openfeature-provider-flagd/tests/e2e/inprocess/flags/allFlags.json

```diff
@@ -0,0 +1,787 @@
+{
```

Comment:
@leakonvalinka @aepfli is this file supposed to be committed? This looks like the e2e flags we provide from the test-harness. I'm not sure why we'd need a copy here.